docs/interpreting-experiments/confidence-intervals.md

```diff
@@ -1,34 +1,266 @@
+
+Once we have an estimate of the observed lift (both its mean and variance), we can use that to construct a confidence interval that describes the plausible values for the true lift.
 
-**Note:** The theoretical guarantees for this method require sub-Gaussian data; which means we have to rely on the central limit theorem and confidence intervals are not accurate in very small sample sizes. We run extensive simulations to validate that this method of generating confidence intervals satisfies the coverage guarantees.
+### Estimating lift
 
-As improvements to the state of the art methods are developed, we will incorporate them into the Eppo platform.
+First, some notation:
 
+* For a metric $X$, we observe $x_i$ for each subject $i$ in the control group of size $N_C$, and $x_j$ for each subject $j$ in the treatment group of size $N_T$.
+* The *true* mean (which we don't know!) of $X$ among the control group is $\mu_C$, and among the treatment group it is $\mu_T$. Similarly, the *true* (unobserved) variance is $\sigma^2_C$ and $\sigma^2_T$
+* $m_C$ and $m_T$ are the averages of $X$ across all subjects in the control group and treatment group, respectively
+* $s^2_C$ and $s^2_T$ are the sample variances of $X$ for the control group and treatment group, respectively
 
-#### Ratio test statistic
-The null hypothesis test, for statistical significance for the ratio between treatment and control, has the form:
+Regardless of the *true* distribution of $X$, the [Central Limit Theorem (CLT)](https://en.wikipedia.org/wiki/Central_limit_theorem) says that the *mean* of $X$, $\bar{X}$, is normally distributed; that is, for the control group:
 
-$H_0: \frac{\hat{\mu}_T}{\hat{\mu}_C} - 1 = 0$
+$$
+\lim_{N_C \to \infty}\bar{X}_C \sim \mathcal{N}(\mu_C; \frac{\sigma^2_C}{N_C})
+$$
 
-The sampling distributions for estimates of the means for control and treatment are both asymptotically Normal following the Central Limit Theorem. We then apply the [Delta method](https://en.wikipedia.org/wiki/Delta_method) to find that the sampling distribution of the ratio statistic is asymptotically normal with variance
+And similarly for the treatment:
+
+$$
+\lim_{N_T \to \infty}\bar{X}_T \sim \mathcal{N}(\mu_T; \frac{\sigma^2_T}{N_T})
+$$
+
+We don't observe the true means and variances, but we *can* estimate them, using $m_C$ and $s_C$ and $m_T$ and $s_T$:
+
+$$
+\lim_{N_C \to \infty}\bar{X}_C \sim \mathcal{N}(m_C; \frac{s^2_C}{N_C});
+
+\lim_{N_T \to \infty}\bar{X}_T \sim \mathcal{N}(m_T; \frac{s^2_T}{N_T})
+$$
+
+We want to calculate the lift:
+
+$$
+\hat{\mathcal{L}} = \frac{\bar{X}_T - \bar{X}_C}{\bar{X}_C} = \frac{\bar{X}_T}{\bar{X}_C} - 1
```

Comment:
Nit: I suggest using $\Delta$ instead of $\mathcal{L}$ since the latter is usually used to denote the likelihood. Delta isn't perfect but a bit more common I think